pkg/services/apiserver/service.go

```diff
@@ -147,11 +147,6 @@ func ProvideService(
 				req.URL.Path = "/"
 			}
 
-			// TODO: add support for the existing MetricsEndpointBasicAuth config option
-			if req.URL.Path == "/apiserver-metrics" {
```

Comment:
@toddtreece -- I think we are avoiding this now, right?

Reply:
yep, nothing depends on this as far as i know